posts/draft.md

```diff
@@ -0,0 +1,174 @@
+# A deep dive into the MyST world
+
+> *This is a series of blog posts inviting you to join me in a little journey I have experienced in the last few weeks at the time to figure it out a nice story for **MyST**
+inside the **Nikola** world.*
+
+Hey, a lot of time without writing, you know... (lack of) time, but let's go back to the
+important things... or at least, interesting stuff to share with you!
+
+Recently, I have been involved with the [EBP](https://executablebooks.org/en/latest/)
+project and community (you can learn more about them [here](https://executablebooks.org/en/latest/about.html)). One of the major projects inside that community is the
+[MyST-Parser](https://github.com/executablebooks/MyST-Parser), a *rich* and *extensible*
+flavor of Markdown meant for technical documentation and publishing. It supports all thesyntax of CommonMark Markdown at its base but, in addition, it includes several
+extensions to CommonMark adding extra syntax features for technical writing, such as the
+roles and directives used by Sphinx in the rST world!
+
+When I looked into this project, I quickly realized it would interesting to explore how
+this flavor of Markdown could potentially integrate with the [Nikola](https://getnikola.com/)
+ecosystem. If you look into my previous blog posts, you will see I am a Nikola
+user and contributor and I am always interested in how we can improve the
+"blogging story" for those using Jupyter Notebooks. And the whole **EBP** project circle
```

Comment:
A meta-question: did you also look into using Jupytext for blogging with MyST Jupyter Notebooks in Nikola? This could be a nice way to keep the structure of notebooks, but with the benefit of version control

Reply:
I briefly look into it but did not go full on that because it was not giving enough exposure to the MyST machinery, IMHO.
I think it would be a really nice topic for another (and related) blog post, actually.